v2/cmd/asoctl/pkg/importresources/import_factory.go

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package importresources
+
+import (
+	"crypto/rand"
+	"math/big"
+	"regexp"
+	"strings"
+	"sync"
+	"unicode"
+
+	"github.com/rotisserie/eris"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	"github.com/Azure/azure-service-operator/v2/internal/set"
+	"github.com/Azure/azure-service-operator/v2/pkg/genruntime"
+)
+
+// importFactory is a helper struct that's used to create things.
+type importFactory struct {
```

Comment:
`ImportHelper` sounds like a better name to me

Reply:
I try to avoid names like `Helper` and `Manager` as they become dumping grounds for too much.

I used factory as it's used for making things, but I'm open to a better suggestion.